applications/fishing-map/src/store.ts

```diff
@@ -34,7 +35,8 @@ const rootReducer = combineReducers({
   resources: resourcesReducer,
   timebar: timebarReducer,
   map: mapReducer,
-  download: downloadReducer,
+  downloadActivity: downloadActivityReducer,
```

Comment:
I not sure if I'd have merge both functionalities in the same slice, any preference?

Reply:
What's the befit of merging them? they need different parameters, initial states and perform different calls